app/_src/mesh/features/provenance-verification-images.md

```diff
@@ -0,0 +1,164 @@
+```
+
+Here's the same example using sample values instead of placeholders:
+
+```sh
+cosign verify-attestation \
+   'kong/kuma-cp:2.7.4@sha256:<manifest_digest>' \
+   --type='slsaprovenance' \
+   --certificate-oidc-issuer='https://token.actions.githubusercontent.com' \
+   --certificate-identity-regexp='^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$'
+```
+
+The command will exit with `0` when the `cosign` verification is completed:
+
+```sh
+...
+echo $?
+0
+```
+
+#### Using slsa-verifier
+
+Run the `slsa-verifier verify-image ...` command:
+
+```sh
+slsa-verifier verify-image \
+   <image>:<tag>@sha256:<manifest_digest> \
+   --print-provenance \
+   --source-uri 'github.com/Kong/<repo>'
+```
+
+Here's the same example using sample values instead of placeholders:
+
+```sh
+slsa-verifier verify-image \
+   'kumahq/kuma-cp:2.7.4@sha256:<manifest_digest>' \
+   --print-provenance \
+   --source-uri 'github.com/kumahq/kuma'
+```
+
+The command will print "Verified SLASA provenance" if successful:
+
+```sh
+...
+PASSED: Verified SLSA provenance
+```
+
+### Complete example
+
+#### Using Cosign
+
+Run the `cosign verify-attestation ...` command:
+
+```sh
+cosign verify-attestation \
+   <image>:<tag>@sha256:<manifest_digest> \
+   --type='slsaprovenance' \
+   --certificate-oidc-issuer='https://token.actions.githubusercontent.com' \
+   --certificate-identity-regexp='^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$' \
+   --certificate-github-workflow-repository='Kong/<repo>' \
+   --certificate-github-workflow-name='<workflow name>' \
+   --certificate-github-workflow-trigger='<workflow trigger>'
+```
+
+Here's the same example using sample values instead of placeholders:
+
+```sh
+cosign verify-attestation \
+   'kumahq/kuma-cp:2.7.4@sha256:<manifest_digest>' \
+   --type='slsaprovenance' \
+   --certificate-oidc-issuer='https://token.actions.githubusercontent.com' \
+   --certificate-identity-regexp='^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$' \
+   --certificate-github-workflow-repository='kumahq/kuma' \
+   --certificate-github-workflow-name='build-test-distribute' \
+   --certificate-github-workflow-trigger='push'
+```
+
+#### Using slsa-verifier
+
+Run the `slsa-verifier verify-image ...` command:
+
+```sh
+slsa-verifier verify-image \
+   <image>:<tag>@sha256:<manifest_digest> \
+   --print-provenance \
+   --source-uri 'github.com/Kong/<repo>' \
+   --source-tag '<release-tag-version>'
+```
+
+Here's the same example using sample values instead of placeholders:
+
+```sh
+slsa-verifier verify-image \
+   'kumahq/kuma-cp:2.7.4@sha256:<manifest_digest>' \
```

Comment:
Provenance is **NOT** generated on `preview/release branches'. Replace `<manifest_digest>` with official image manifest digest of `2.7.4` to verify the provenance